certbot-apache/certbot_apache/tests/augeas_configurator_test.py

```diff
@@ -33,12 +33,12 @@ def test_bad_parse(self):
         self.config.parser.parse_file(os.path.join(
```

Comment:
I think we should rename this file and update comments and names. Maybe something like `configurator_reverter_test`?